azurelinuxagent/ga/exthandlers.py

```diff
@@ -98,14 +98,16 @@ class ValidHandlerStatus(object):  # pylint: disable=R0903
 _EXTENSION_TERMINAL_STATUSES = [ValidHandlerStatus.error, ValidHandlerStatus.success]
 
 
+# too-few-public-methods<R0903> Disabled: This class is used as an Enum
 class ExtCommandEnvVariable(object):  # pylint: disable=R0903
```

Comment:
Do you need `# pylint: disable=R0903` here if you suppressed it above the class?

Reply:
Nevermind, the comment is just a clarification, not a suppression.